src/Form.js

```diff
@@ -34,10 +35,20 @@ function Form(props) {
     });
   };
 
+  const validateSubmit = (e) => {
+    if (formData.name == "" || formData.date == "" || formData.note == "") {
+      e.preventDefault();
+    }
+  };
+
+  const handleSubmit = (e) => {
+    validateSubmit(e);
```

Comment:
e.preventDefault() should go here on line 45 above validateSubmit()

Reply:
Instead of just disabling the button depending on if some of the fields are required or not, you need to generate logic that will display a message to the user specifying that the fields are required.

Also you have a function within a function here...
Example:
```
const validateSubmit = () => {
  // validation logic
  // check through each of the formData values if they are valid or not
  // update the formErrors state to display the message
  // return if it is validated or not
}

const handleSubmit = (e) => {
  e.preventDefault();
  const isValidated = validateSubmit();
  if (isValidated) {
    // logic to submit a form and update notes state
  }
}
```